tracee-rules/signature.go

```diff
@@ -97,7 +98,16 @@ func findRegoSigs(dir string) ([]types.Signature, error) {
 		}
 		sig, err := regosig.NewRegoSignature(string(regoCode), regoHelpersCode)
 		if err != nil {
-			log.Printf("error creating rego signature with: %s: %v ", regoCode[0:20], err)
+			newlineOffset := bytes.Index(regoCode, []byte("\n"))
+			if newlineOffset == -1 {
+				codeLength := len(regoCode)
+				if codeLength < 22 {
+					newlineOffset = codeLength
+				} else {
+					newlineOffset = 22
+				}
```

Comment:
newlineOffset = 
```suggestion
                              newlineOffset = len(regoCode) % 22 
```

Reply:
problem is if the length of the code is exactly 22 characters - the message won't print anything...

Reply:
Ah makes sense, good point